.ci/setup.sh

```diff
@@ -39,4 +51,20 @@ then
         --body-length 72 \
         --subject-length 75 \
         --verbose
+    # Travis doesn't provide a VT-x environment, so nothing more to do
+    # here.
+    exit 0
 fi
+
+# Check agent builds
+cd go get -d ${github_project} || true
```

Comment:
Oh and replace your command here:
s/cd go get -d ${github_project} || true/go get -d ${github_project} || true/
You've added one `cd` that we don't need.